.github/workflows/ruby.yml

```diff
@@ -0,0 +1,27 @@
+name: Ruby
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  Build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@359bebbc29cbe6c87da6bc9ea3bc930432750108
+        with:
+          ruby-version: '3.1'
+
+      - name: Install dependencies
+        run: bundle install
+        
+      - name: Run tests
+        run: bundle exec rspec
```

Comment:
Could you also add a step for running a linter in CI?

Reply:
added!